package.json

```diff
@@ -99,10 +99,10 @@
     "check-types:frontend": "tsc --noEmit --project .",
     "depcheck": "check-node-version --node \">= 12.0\"",
     "prebuild": "run-s --silent depcheck",
-    "features": "node scripts/export-supported-features-cli.js > supported-features.json",
-    "defs": "node scripts/export-service-definitions-cli.js > service-definitions.yml",
-    "build": "run-s defs features && gatsby build",
-    "heroku-postbuild": "run-s --silent build",
+    "features": "node scripts/export-supported-features-cli.js > ./frontend/supported-features.json",
+    "defs": "node scripts/export-service-definitions-cli.js > ./frontend/service-definitions.yml",
+    "build": "run-s defs features && cd frontend && gatsby build",
+    "heroku-postbuild": "run-s --silent build && mv frontend/public .",
```

Comment:
Neither Gatsby nor Heroku support custom folders, it must be `public` in the directory the command is run for Gatsby and  `public` in the project root for Heroku. Manually moving to align with Heroku's requirements.

https://github.com/heroku/heroku-buildpack-nodejs/issues/385
https://github.com/gatsbyjs/gatsby/discussions/1878